packages/block-editor/src/hooks/use-bindings-attributes.js

```diff
@@ -115,110 +115,117 @@ export const withBlockBindingSupport = createHigherOrderComponent(
 		// there are attribute updates.
 		// `source.getValues` may also call a selector via `registry.select`.
 		const boundAttributes = useSelect( () => {
-			if ( ! bindings ) {
+			if ( ! blockBindings ) {
 				return;
 			}
 
 			const attributes = {};
 
-			for ( const [ attributeName, boundAttribute ] of Object.entries(
-				bindings
+			const bindingsBySource = new Map();
```

Comment:
Should this be `blockBindingsBySource` to keep consistency?

Reply:
I have just changed this 🙂